src/polyswarm_api/utils.py

```diff
@@ -55,3 +56,14 @@ def parse_hashes(hashes, hash_type=None, hash_file=None):
         except InvalidHashException:
             logger.warning("Invalid hash %s provided, ignoring.", h)
     return out
+
+
+def try_import(name):
```

Comment:
I like this, but I think we should have an `extras.py` module or even an `extras` package if we want to be future ready and we envision having many plugable features.

Reply:
Hrm, a good thought. I don’t know how much Beyond the features extraction would require this but it’s an idea.